### PR TITLE
Chore: Output rendered queries to debug log.

### PR DESCRIPTION
**Pull request by crericha**

### CODE REVIEW

File: sqlmesh/utils/formatting.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+def indent(string: str, num: int = 1) -> str:
```
Comment (tobymao) [inline sqlmesh/utils/formatting.py:1]: why do you need this?

Comment (crericha) [reply sqlmesh/utils/formatting.py:1]: Its difficult to scan the log when the query is inline with all the other text. Indenting made it easy to parse.

Comment (tobymao) [reply sqlmesh/utils/formatting.py:1]: this only indents line breaks though, can you just do

query.sql(indent=4)

Comment (crericha) [reply sqlmesh/utils/formatting.py:1]: Didn't realize sql() had an indent option. Will look into that.

Comment (crericha) [reply sqlmesh/utils/formatting.py:1]: @tobymao what I'm trying to do is indent level 0. `indent` only applies to level 1+.

Comment (crericha) [reply sqlmesh/utils/formatting.py:1]: Screenshot 2023-05-31 at 9 51 26 PM

Comment (tobymao) [reply sqlmesh/utils/formatting.py:1]: ok, use textwrap.indent

**Review by tobymao — APPROVED** [review]

**Pull request merged**